### DISCUSSION TITLE
Manipulated Variable blocking

### DISCUSSION THREAD

**Discussion by caxelrud** [franckgaga/ModelPredictiveControl.jl#127] [General]
Hi,
I didn't see the use of the Manipulated Variable blocking in ModelPredictiveControl.jl.
This feature is common in MPCs, mainly to deal with long horizons.
For Matlab MPC, it is defined at [MV blocking](https://www.mathworks.com/help/mpc/ug/manipulated-variable-blocking.html).
Let me know your opinion.

**Comment by franckgaga** [comment]
True! It was in one of the next item in my list. I need to finish custom nonlinear constraint in `NonLinMPC` first.

Good to see you back @caxelrud! :)

**Comment by franckgaga** [comment]
The next release will introduce this feature!
